packages/core/src/forms/control-action/control-action.element.ts

```diff
@@ -4,89 +4,31 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import { html } from 'lit';
-import { CdsIcon } from '@cds/core/icon/icon.element.js';
-import {
-  property,
-  assignSlotNames,
-  baseStyles,
-  CdsBaseButton,
-  hasAttributeAndIsNotEmpty,
-  querySlot,
-  setOrRemoveAttribute,
-} from '@cds/core/internal';
-import styles from './control-action.element.scss';
-import { LogService } from '@cds/core/internal';
+import { PropertyValues } from 'lit';
+import { assignSlotNames, property } from '@cds/core/internal';
+import { CdsButtonAction } from '@cds/core/button-action';
 
 /**
- * Control Action
+ * Control Action Button
  *
  * ```typescript
- * import '@cds/core/forms/register.js';
+ * import '@cds/core/button/register.js';
```

Comment:
Looking at the code, I'm not sure where control-action needs to be imported from but I'm 100% sure it's not being exported from `button/register.ts`...

Reply:
Yeah will fix, that looks like I copy/pasted to much

Reply:
fixed